PRINCIPLES.md

```diff
@@ -15,22 +16,32 @@ Approved by GB on: Dec 5, 2017
 
 ## TOC Operating Principles
 
-Now that CNCF has been active for over a year we want to start writing down what we have learned. Future TOCs can still make changes, but there will at least be documented precedent.
+ As part of our continued community engagement and iterations on the work done, the following list of principles serve to provide an understanding of how the TOC intends to work with projects, effectively the guiding star the the TOC in their work. It also provides insight into what the TOC is looking for continuity and evolution across each seated TOC. Since the CNCF's inception we have endevoured to capture the lessons learned and interate on these principles to service the foundation, the community, and the projects.
```

Comment:
`As part of our continued community engagement, the following list of principles serve to provide an understanding of how the TOC intends to work with projects, effectively serving as the guiding star for the TOC. It also provides insight into what the TOC is looking for in terms of continuity and evolution over time. Since the CNCF's inception we have endeavoured to capture the lessons learned and iterate on these principles to serve the foundation, the community, and the projects.`